frontend/src/Explorer/View/Address.purs

```diff
@@ -29,53 +31,57 @@ addressView state =
             [ P.div
                   [ P.className "explorer-address__container" ]
                   [ P.h3
-                          [ P.className "headline"]
-                          [ P.text $ translate (I18nL.common <<< I18nL.cAddress) lang' ]
+                        [ P.className "headline"]
+                        [ P.text $ translate (I18nL.common <<< I18nL.cAddress) lang' ]
                   , addressOverview (state ^. currentAddressSummary) lang'
                   ]
             ]
             , P.div
                 [ P.className "explorer-address__wrapper" ]
                 [ P.div
                       [ P.className "explorer-address__container" ]
-                      [ P.h3
-                          [ P.className "headline"]
-                          [ P.text $ translate (I18nL.common <<< I18nL.cTransactions) lang' ]
-                        , case state ^. currentAddressSummary of
-                              NotAsked  -> emptyTxHeaderView
-                              Loading   -> emptyTxHeaderView
-                              Failure _ -> emptyTxHeaderView
-                              Success (CAddressSummary addressSummary) ->
-                                  let txList = addressSummary ^. caTxList
-                                      txPagination = state ^. (viewStates <<< addressDetail <<< addressTxPagination)
-                                      currentTxBrief = txList !! (txPagination - 1)
-                                  in
-                                  P.div
-                                      []
-                                      [ txHeaderView lang' $ case currentTxBrief of
-                                                                Nothing -> mkTxHeaderViewProps mkEmptyViewProps
-                                                                Just txBrief -> mkTxHeaderViewProps txBrief
-                                      , txBodyView $ case currentTxBrief of
-                                                                  Nothing -> mkTxBodyViewProps mkEmptyViewProps
-                                                                  Just txBrief -> mkTxBodyViewProps txBrief
-                                      , txPaginationView
-                                            { label: translate (I18nL.common <<< I18nL.cOf) $ lang'
-                                            , currentPage: txPagination
-                                            , maxPage: length txList
-                                            , changePageAction: AddressPaginateTxs
-                                            , onFocusAction: SelectInputText <<< targetToHTMLInputElement
-                                            }
-                                      ]
-
-                      ]
+                      [ case state ^. currentAddressSummary of
+                            NotAsked  -> emptyAddressTxView
+                            Loading   -> emptyAddressTxView
```

Comment:
@ksaric It's already there within the `addressOverview` located above (in `line 83` https://github.com/input-output-hk/cardano-sl-explorer/blob/fix/CSE-70-search-invalid-state/frontend/src/Explorer/View/Address.purs#L83)